sql/fxa_users_daily_v1.sql

```diff
@@ -42,7 +42,8 @@ WITH
     user_id,
     ROW_NUMBER() OVER w1_unframed AS _n,
     udf_mode_last(ARRAY_AGG(country) OVER w1) AS country,
-    udf_contains_tier1_country(ARRAY_AGG(country) OVER w1) AS seen_in_tier1_country
+    udf_contains_tier1_country(ARRAY_AGG(country) OVER w1) AS seen_in_tier1_country,
+    EXISTS(SELECT event_type FROM ARRAY_AGG(event_type) WHERE event_type = 'fxa_reg - complete') AS registered
```

Comment:
@irrationalagent I'm making a guess here than an event with `event_type = 'fxa_reg - complete'` is our indicator of creation of a new Firefox Account. Is that even close to correct? Do you have a suggestion for how we define new account creation?

Reply:
yep, that is the best way to flag a new account. FYI, `fxa_login - complete` reflects existing users logging into their account.